linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_networkconfig_modification/rule.yml

```
@@ -41,7 +41,7 @@ references:
     srg@rhel6: SRG-OS-999999
     nist@rhel6: AC-3(10)
     nist-csf@rhel6: PR.AC-4,PR.AC-6,PR.PT-3
-    cis: 5.2.6
+    cis@rhel8: 4.18
```

```suggestion
    cis@rhel8: "4.18"
```
The build system interprets as a float number in this case. It needs to be double quoted

---

Ack & fixed.Thansk!